src/clients/node/scripts/build.sh

```diff
@@ -1,9 +1,20 @@
 #!/usr/bin/env bash
+# This script builds our release node package. It pulls in
+# node:14-buster specifically, to be on an old-ish version of glibc
+# with the minimum node version we support for building.
+#
+# Both glibc and NAPI have backward-compatible ABIs
+set -e
 
- set -e
+docker run -v "$(pwd)/../../..":/wrk -w /wrk --entrypoint bash node:14-buster -c '
```

Comment:
Dumb question :D I don't understand the changes here. What do these mean?

Reply:
So before, `build.sh` was getting invoked each time to generate the package that was being used by the distro tests.

Now, `build.sh` actually builds the final tarball, and it builds it once, which then gets tested by everything and published if on main (this part pending, still need to push the publish changes).

It does this by building specifically on `node:14-buster` within Docker - being the oldest version of Node we support, and an older Debian, to ensure that both glibc and NAPI don't somehow pull in any symbols that are only available in newer versions (they both give binary ABI backwards compatibility guarantees).

The first 2 lines of the script are just the header downloading script, condensed, since we fully control the environment the CI build runs in.